modules/assim.sequential/R/sda.enkf.R

```diff
@@ -83,19 +96,11 @@ sda.enkf <- function(settings, IC, prior, obs.mean, obs.cov, variables,
     print(paste("please make sure that the PEcAn interface is loaded for",model))
     stop()
   }
-  
-  ## split clim file
-  ## leaving in for sipnet
-  #   full.met <- settings$run$inputs$met$path
-  #   new.met  <- file.path(settings$rundir,basename(full.met))
-  #   file.copy(full.met,new.met)
-  #   met <- new.met#split.met.SIPNET(new.met)
-  
-  
+ 
   ###-------------------------------------------------------------------###
   ### perform initial set of runs                                       ###
   ###-------------------------------------------------------------------###  
-  X = IC
+  X = matrix(NA,as.numeric(settings$ensemble$size),length(settings$pft)) #why does X need to be here?
```

Comment:
Start by reverting to the previous workflow -- for now take IC as a function argument and use the existing sipnet IC function to generate that IC. When using LINKAGES you can default IC to NULL since you don't use it